aten/src/ATen/native/native_functions.yaml

```diff
@@ -5699,6 +5699,10 @@
     CPU: _svd_helper_cpu
     CUDA: _svd_helper_cuda
 
+- func: swapdim(Tensor(a) self, int dim1, int dim2) -> Tensor(a)
```

Comment:
"swapdims"

Please also add "swapaxes," both as aliases of transpose. Instructions for adding aliases are here:

https://github.com/pytorch/pytorch/blob/a814231616f5db7fcdb8793f64c12d652d1e8572/aten/src/ATen/native/native_functions.yaml#L232

Reply:
(You don't have to implement the aliases for the named variant of transpose).